src/Command/AddMassMediaCommand.php

```diff
@@ -22,11 +25,26 @@
  */
 class AddMassMediaCommand extends BaseCommand
 {
+    /**
+     * @var Registry|null
+     */
+    protected $doctrine;
+
     /**
      * @var string[]
      */
     protected $setters;
 
+    public function __construct(ManagerInterface $mediaManager, Pool $pool, ?Registry $doctrine = null)
+    {
+        $this->doctrine = $doctrine;
+
+        parent::__construct($mediaManager, $pool);
```

Comment:
isnt it better to call first parent and the do the missing stuff?

Reply:
Done